cypress/support/commands.ts

```diff
@@ -73,7 +73,28 @@ Cypress.Commands.add(
       path: `/activityhistory?targetId=${targetId}&pageSize=100`,
       status: 200,
       body: {
-        results: [{}],
+        results: [
+          {
+            id: 'b0058e24-ef9c-4672-a830-dc5c1a329dce',
+            targetId: targetId,
+            type: 'update',
+            targetType: 'housingApplication',
+            createdAt: '2024-09-04T13:13:59.7981926Z',
+            timetoLiveForRecord: 0,
+            oldData: {
+              'assessment.biddingNumber': null,
+            },
+            newData: {
+              _activityType: 'biddingNumberChangedByUser',
+              'assessment.biddingNumber': '1000001',
+            },
+            authorDetails: {
+              fullName: 'John Jones',
+              email: 'test@hackney.gov.uk',
+            },
+            sourceDomain: 'HousingRegister',
+          },
+        ],
```

Comment:
I think you'll need to adjust this, as it's currently used across the tests with empty response.

So you could just adjust it so you can pass a result to it, or otherwise it will be an empty object.